gapic-generator-java/DEVELOPMENT.md

```diff
@@ -155,6 +155,39 @@ To generate a production GAPIC API:
    ```
    and then rebuild gapic-generator-java (`mvn clean install`).
 
+## Debugging the Plugin under googleapis with gapic-generator-java
+
+To debug the generator starting from protobuf compiler plugin
+
+1. In [googleapis](https://github.com/googleapis/googleapis) root directory, run the following commands:
+   1. Set `JVM_DEBUG_PORT` environment variable.
+       ```shell
+       export JVM_DEBUG_PORT=5005
+       ```
+      The proto compiler runs the `protoc-gen-java_gapic` shell script (a wrapper of java command with an option to specify JVM debugging). 
+      We set the environment variable `JVM_DEBUG_PORT` to initialize the debug mode.
```

Comment:
The relationship between `protoc-gen-java_gapic` and `JVM_DEBUG_PORT` is unclear

Reply:
How about

We set the environment variable `JVM_DEBUG_PORT` to enable the debug mode.

Reply:
“to enable the debug option”

(“option” is the word used in the previous sentence)

Reply:
done